app/javascript/packs/image-preview.js

```diff
@@ -26,4 +26,58 @@ function imagePreview() {
   });
 }
 
+function frontImagePreview() {
+  $('#take_picture').on('click', function() {
+    document.getElementById('doc_auth_front_image').click();
+  });
```

Comment:
I haven't tested to see how this behaves, but seeing as the same ID is being used for both the front and back image, each with a click handler to click on a different element, I've got to imagine there would be some conflict, where both would be clicked.